src/views/Events/index.js

```diff
@@ -254,68 +288,63 @@ export default class Events extends Component {
       content = <EventList events={this.state.filteredEvents} />;
     }
 
+    let eventsAtGordon = [];
```

Comment:
You need to use state to store this variable. Otherwise, it's getting recalculated every time the component renders, and so will be almost constantly empty. Also, you could probably rename this to `activeFilters`, since that's what it represents.

Reply:
https://github.com/gordon-cs/gordon-360-ui/pull/1015/commits/0990cff23f671494e1aa8835dfacbca3d2b2b1d6